FabricExample/App.tsx

```diff
@@ -10,6 +10,7 @@ import AnimatedSensorExample from './src/AnimatedSensorExample';
 import AnimatedTextInputExample from './src/AnimatedTextInputExample';
 import AnimatedTextWidthExample from './src/AnimatedTextWidthExample';
 import ArticleProgressExample from './src/ArticleProgressExample';
+import BillardExample from './src/BillardExample';
```

Comment:
There's a spelling mistake. It is missing an "i".

The correct spelling is Billiard

Reply:
Thanks! Fixed in d33ee7221fa98e7fe49519803b6a8060256cc1a7